### ISSUE TITLE
VCA envelope tails are a bit abrupt

### ISSUE THREAD

**Issue by JordanAceto** [JordanAceto/josh_Ox_ribbon_synth#17]
When the VCA envelope is decaying, the sound does not decay as smoothly as we would like.

This is due to resistor R407 on the VCF/VCA board, which ensures that when zero volts is applied to the CV input, the output signal is completely silenced. It "pushes down" the CV input by a tiny bit.

But if too much current flows through this resistor, it can make the long tails of the volume envelope end abruptly. The final decay doesn't sound as smooth as it should.

Right now the volume pot turns up and down the control voltage that goes to the final VCA. This means when the volume is turned down low, the ADSR signal is small relative to the impact of the "push-down" resistor R407. If instead we always sent the ADSR signal to the VCA CV full-strength, and then attenuated the signal afterwards, I think the un-smooth decay would effect would be reduced.

Since the volume knob is way over on the Ribbon board, we should probably add another VCA just for the master volume. The Ribbon board can send two VCA signals, one is the volume ADSR, and the other is a [0, 5v] master volume control voltage. The final master volume VCA could be exponential control (IE a single SSI216x section plus an opamp), so we can get a nice audio taper from a linear potentiometer on the Ribbon board.

TL;DR add an expo control VCA after the main VCA. The first VCA gets the volume ADSR as a control signal, the final VCA is controlled from the master volume pot.

**Comment by JordanAceto** [comment]
The new ADSR and VCA setup fixes this.
